cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/util/ClouderaManagerCommandApiErrorParserUtil.java

```diff
@@ -0,0 +1,68 @@
+package com.sequenceiq.cloudbreak.cm.util;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.cloudera.api.swagger.CommandsResourceApi;
+import com.cloudera.api.swagger.client.ApiException;
+import com.cloudera.api.swagger.model.ApiCommand;
+import com.cloudera.api.swagger.model.ApiCommandList;
+
+public class ClouderaManagerCommandApiErrorParserUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClouderaManagerCommandApiErrorParserUtil.class);
+
+    private ClouderaManagerCommandApiErrorParserUtil() {
+
+    }
+
+    private static Predicate<ApiCommand> commandFailed() {
+        return cmd -> {
```

Comment:
This predicate might be better readable on what it does is to have a separate filter for null checks.